authentication/src/main/java/io/camunda/authentication/config/WebSecurityConfig.java

```diff
@@ -34,6 +38,12 @@ public class WebSecurityConfig {
       new String[] {"/login**", "/logout**", "/error**", "/actuator**"};
   private static final Logger LOG = LoggerFactory.getLogger(WebSecurityConfig.class);
 
+  @Bean
+  public MethodSecurityExpressionHandler methodSecurityExpressionHandler(
```

Comment:
🔧 When I tried to start the code locally with the profiles `operate, identity, broker, auth-basic` Spring complained of multiple beans, with this change we should make sure that the old classes are also either updated to be set on profiles or maybe this one is primary.

It may also be that with this change and for 8.7 the existing method security classes can be removed actually, maybe someone from one of the component teams or API team could confirm (@tmetzke do you have thoughts?)